mne/preprocessing/ica.py

```diff
@@ -2624,3 +2638,39 @@ def corrmap(icas, template, threshold="auto", label=None, ch_type="eeg",
         return template_fig, labelled_ics
     else:
         return None
+
+
+def read_eeglab_ica(file_name):
+    """Load ica information saved in an EEGLAB .set file.
+
+    Parameters
+    ----------
+    file_name : Complete path to a .set EEGLAB file that contains an ica.
+
+    Returns
+    -------
+    ica : A mne.preprocessing.ICA object based on the information
```

Comment:
Please  check how we format docstrings

Reply:
I guess you are referring to the argument types that were missing. This has been corrected.